### PR TITLE
Day one

### PR DESCRIPTION
**Pull request by Munsa1**
In the following pull request, we have:

-Created three files,  one html, one CSS and and JavaScript file
-Used a form in html to create a simple and plain user interface
-Used JavaScript to implement the requirements

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  books = books.filter((y) => y !== books[books.findIndex((x) => x.id === parseInt(buttonId, 10))]);
+  localStorage.setItem('bookObject', JSON.stringify(books));
+  document.getElementById('Result').innerHTML = `${books.map(BookList).join('')}`;
+};
+// Add Form data to Unordered List
+const addBooks = (ev) => {
+  ev.preventDefault();
+  i += 1;
+  const book = {
+    id: i,
```
Comment (Davidosky007) [inline index.js:22]: * I think you should make your id unique currently when the remove button is clicked on identical id numbers both are removed. please kindly fix this 🙏 as it deletes the wrong book

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @sheylaPozo @Munsa1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly check the comments and make the necessary changes 👍 
# _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.js (hunk 1 above)
Comment (Munsa1) [reply index.js:22]: I fixed it

Comment (Davidosky007) [reply index.js:22]: nice 👍

**Comment by Davidosky007** [comment]
Hi @Munsa1 @sheylaPozo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# [OPTIONAL]
* this is an Optional suggestion and not a requirement for this project but would be a nice to have feature 
* I think you can clear the input field after a book is created so that creation of new books would be easy 👍  
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**